### ISSUE TITLE
New Notebook - prompts for kernel selection

### ISSUE THREAD

**Issue by jjbursik** [krassowski/jupyterlab-voice-control#5]
First, this is really cool and a great idea!

My issue, is more a user question probably but it gets to adding custom commands. When I give the command for "new notebook", Jupyter will create a new `.ipynb` file but then a pop up is given for me to select the type of kernel I want the notebook to work with. I couldn't figure out which voice command to provide to select the type of kernel. Is this an option?

If this isn't an option today, how could I create the custom voice command to work around this?

**Comment by krassowski** [comment]
Thanks for opening the issue. Interacting with dialog windows is definitely in scope. I am not sure if we should add some commands to JupyterLab itself (it might be useful, as we do interact with dialogs in UI tests too), or just add commands in this extension directly. Either way we can start by adding a command here and then move it to JupyterLab itself if there is an agreement on this.

**Comment by jjbursik** [comment]
@krassowski yeah, that makes sense to me! I'm good with starting here and then moving it to Lab later.
